File: Testing/MyUnitTesting/java/app/src/main/java/com/dicoding/picodiploma/myunittest/CuboidModel.java
package com.dicoding.picodiploma.myunittest;

// Cuboid == Balok
class CuboidModel {
    private double length;
    private double width;
    private double height;

    public CuboidModel() {
    }

    public void save(double length, double width, double height) {
        this.length = length;
        this.width = width;
        this.height = height;
    }

    double getVolume() {
        return length * width * height;
    }

    public double getSurfaceArea() {
        double lw = length * width;
        double wh = width * height;
        double lh = length * height;

        return 2 * (lw + wh + lh);
    }

    public double getCircumference() {
        return 4 * (length + width + height);
    }
}

File: Testing/MyUnitTesting/kotlin/app/src/main/java/com/dicoding/picodiploma/myunittest/CuboidModel.kt
package com.dicoding.picodiploma.myunittest

// Cuboid == Balok
class CuboidModel {
    private var length: Double = 0.0
    private var width: Double = 0.0
    private var height: Double = 0.0

    fun save(length: Double, width: Double, height: Double) {
        this.length = length
        this.width = width
        this.height = height
    }

    fun getVolume(): Double = length * width * height

    fun getSurfaceArea(): Double {
        val lw = length * width
        val wh = width * height
        val lh = length * height

        return 2 * (lw + wh + lh)
    }

    fun getCircumference(): Double = 4 * (length + width + height)
}